eosdis-nasa/earthdata-pub-dashboard
# EDPUB-1401 & EDPUB-1402: Create/Edit/List Dashboard page for a section

# Description

Edpub 1401 & EDPUB-1402: Create/Edit Dashboard page for a section

## Spec

See Ticket: [EDPUB-1402](https://bugs.earthdata.nasa.gov/browse/EDPUB-1402)
[EDPUB-1401](https://bugs.earthdata.nasa.gov/browse/EDPUB-1401)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Navigate to **base_url/sections** -> Add/Edit section to check if it works as expected
